src/phylo/distances.jl

```diff
@@ -0,0 +1,172 @@
+# phylo/distances.jl
+# ==================
+#
+# Types and methods for computing evolutionary and genetic distances.
+#
+# Part of the Bio.Phylo module.
+#
+# This file is a part of BioJulia. License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+# Types
+# -----
+
+# Evolutionary distances
+abstract EvoDist
+abstract UncorrectedDist <: EvoDist
+abstract CorrectedDist <: EvoDist
+abstract TsTv <: CorrectedDist
+
+immutable N_Mutations{T} <: UncorrectedDist end
+immutable P_Distance{T} <: UncorrectedDist end
```

Comment:
Is there any references for `N_Mutations` and `P_Distance`? Also, underscore in a type name looks a little bit unusual in Julia.